### ISSUE TITLE
Any PyTorch implementation available

### ISSUE THREAD

**Issue by Ivan-Fan** [diheal/negative#1]
Hi! Very nice work! Really like the negative sampling idea in the paper~

I was wondering, do we have a pytorch version for the code if possible? Since I saw the pytorch-like pseudocode in the paper. Thanks!

**Comment by jqwangai** [comment]
Sorry, no Pytorch code is available. 

But you can refer to this link https://github.com/Tian0426/CL-HAR, to replace the loss function.

**Comment by Ivan-Fan** [comment]
Thank you!
